xwiki-enterprise-ui/xwiki-enterprise-ui-mainwikipreferences/src/main/resources/XWiki/XWikiPreferences.xml

```diff
@@ -2781,7 +2781,7 @@ Your account on ${wikiname} has been successfully activated. You can now login a
       <registration_registered>---</registration_registered>
     </property>
     <property>
-      <rightPanels>Panels.QuickLinks,Panels.MyRecentModifications</rightPanels>
+      <rightPanels>Panels.MyRecentModifications</rightPanels>
```

Comment:
An idea is to introduce a tip panel and have one tip about ctrl+g (that you've lost with your change). With a few tips to start with, displayed at random.